File: php/app/Http/Controllers/Manage/PlayerController.php
<?php

namespace TrentonDarts\Http\Controllers\Manage;

use Illuminate\Http\Request;

use TrentonDarts\Http\Requests;
use TrentonDarts\Http\Controllers\Controller;

use TrentonDarts\Http\Requests\StoreLeagueManagementRequest;

use TrentonDarts\User;
use TrentonDarts\LeagueManagement\Models\Player;

class PlayerController extends Controller
{
    /**
     * Display a listing of the resource.
     *
     * @return Response
     */
    public function index($leagueId)
    {
        $players = Player::where('leagueId', $leagueId)->orderBy('firstName')->get();
        return view('manage.player.list', ['leagueId' => $leagueId, 'players' => $players]);
    }

    /**
     * Show the form for creating a new resource.
     *
     * @return Response
     */
    public function create($leagueId)
    {
        $users = User::orderBy('name')->lists('name', 'id');
        return view('manage.player.create', ['leagueId' => $leagueId, 'users' => $users]);
    }

    /**
     * Store a newly created resource in storage.
     *
     * @param  Request  $request
     * @return Response
     */
    public function store($leagueId, StoreLeagueManagementRequest $request)
    {
        $this->validate($request, [
            'email' => 'email|unique:players|max:255'
        ]);

        $player = Player::create($request->input());
        $player->leagueId = $leagueId;
        $player->save();

        return redirect()->route('manage.player.index', ['leagueId' => $leagueId]);
    }

    /**
     * Display the specified resource.
     *
     * @param  int  $id
     * @return Response
     */
    public function show($id)
    {
        //
    }

    /**
     * Show the form for editing the specified resource.
     *
     * @param  int  $id
     * @return Response
     */
    public function edit($leagueId, $id)
    {
        $player = Player::find($id);
        $users = User::orderBy('name')->lists('name', 'id');
        return view('manage.player.edit', ['leagueId' => $leagueId, 'player' => $player, 'users' => $users]);
    }

    /**
     * Update the specified resource in storage.
     *
     * @param  Request  $request
     * @param  int  $id
     * @return Response
     */
    public function update($leagueId, $id, StoreLeagueManagementRequest $request)
    {
        $this->validate($request, [
            'email' => 'email|unique:players,email,'.$id.'|max:255'
        ]);
        $player = Player::findOrFail($id);
        $player->update($request->input());
        $player->save();
        return redirect()->route('manage.player.index', ['leagueId' => $leagueId]);
    }

    /**
     * Remove the specified resource from storage.
     *
     * @param  int  $id
     * @return Response
     */
    public function destroy($leagueId, $id)
    {
        $player = Player::findOrFail($id);
        $player->delete();

        return redirect()->route('manage.player.index', ['leagueId' => $leagueId]);
    }
}

File: src/DartLeague/DartLeague.Web/Areas/Manage/Controllers/PlayerController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using DartLeague.Web.Areas.Manage.Models;
using Microsoft.AspNetCore.Mvc;
using Microsoft.AspNetCore.Mvc.Routing;

namespace DartLeague.Web.Areas.Manage.Controllers
{
    [Area("Manage")]
    public class PlayerController : Controller
    {
        public IActionResult Index()
        {
            return View();
        }
        public IActionResult Create()
        {
            var model = new Player();
            return View(model);
        }

        [Route("Manage/Player/Edit/{id}")]
        public IActionResult Edit(string id)
        {
            return View(new Player(){FirstName = $"My ID is{id}"});
        }

        public IActionResult List()
        {
            return View(new PlayersList
            {
                Players = new List<Player> {
                new Player {FirstName = "Larry", ID = "1"},
                new Player {FirstName = "Moe", ID ="2"},
                new Player {FirstName = "Curly", ID = "3"}
                }
            });
        }

        [HttpPost]
        public IActionResult Create(Player player)
        {
            return Redirect("Index");
        }

        [HttpPost]
        public IActionResult Edit(Player player)
        {
            return Redirect("Index");
        }

        public IActionResult Delete()
        {
            return Redirect("List");
        }

        
    }
}